### PR TITLE
toLowerCase and toUpperCase with English.

### PR DESCRIPTION
**Pull request by portlek**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing
[pull-request]: https://github.com/JDA-Applications/JDA-Utilities/pulls

#### Pull Request Checklist
Please follow the following steps before opening this PR.<br>
PRs that do not complete the checklist will be subject to denial for
missing information.

- [x] I have checked the [pull request page][pull-request] for upcoming
      or merged features/bug fixes.
- [x] I have read JDA's [contributing guidelines][contributing].

#### Pull Request Information
Check and fill in the blanks for all that apply:

- [x] My PR fixes a bug, error, or other issue with the library's codebase.
- [x] My PR is for the `all` module of the JDA-Utilities library.

#### Description

PR fixes some language bugs when using toLowerCase and toUpperCase methods. For example Turkish, `i.toUpperCase()` will be `İ` which complains Java because it's not a English letter, or `I.toLowerCase()` will be `ı`, again not a English letter.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -51,8 +51,8 @@ allprojects {
             json = { [group: 'org.json', name: 'json', version: jsonVersion] }
             junit = { [group: 'junit', name: 'junit', version: junitVersion] }
 
-            artifactId = (rootProject == project? project.name : "$rootProject.name-$project.name").toLowerCase()
-            moduleName = "${group}.jdautilities${rootProject == project? "" : ".${project.name.toLowerCase()}"}"
+            artifactId = (rootProject == project? project.name : "$rootProject.name-$project.name").toLowerCase(Locale.ENGLISH)
```
Comment (Andre601) [inline build.gradle]: I think `Locale.ROOT` would be better here.

Comment (portlek) [reply build.gradle]: correct.

**Review by Chew — APPROVED** [review]
Seems fine, I suppose. More benefits here than cons?

**Comment by portlek** [comment]
> Seems fine, I suppose. More benefits here than cons?

there are no cons of this PR so, yes more benefits.

**Pull request merged**